common/src/test/java/org/jboss/pnc/common/json/moduleconfig/JenkinsBuildDriverModuleConfigTest.java

```diff
@@ -10,12 +10,11 @@
     @Test
     public void test() {
```

Comment:
I didn't write it so I just changed what was needed, but you are right. I'll fix it